db/migrate/20211211082531_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The migration is fine except for one thing. The user should not be directly referenced to the post object instead with a different name (`author`) than the model name so that I can get the author of a post using the syntax `post_instance.author` instead of `post_instance.user`. To do so the migration should have something like this. 
```
`t.references :author, foreign_key: { to_table: 'users' }`
```
Kindly use the proper migration to remove the current migration and add another one. Do not directly edit migration files. :pray: 
Please carefully look at the ERD. 
https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png